microsoft/testsuites/power/common.py

```diff
@@ -150,3 +171,34 @@ def cleanup_env(environment: Environment) -> None:
         kill.by_name("iperf3")
         kill.by_name("fio")
         kill.by_name("stress-ng")
+
+
+def _expand_os_partition(node: Node, log: Logger) -> None:
+    if isinstance(node.os, Redhat):
+        pv_result = node.execute("pvscan -s", sudo=True, shell=True).stdout
+        # The output of pvscan -s is like below.:
+        #  /dev/sda4
+        #  Total: 1 [299.31 GiB] / in use: 1 [299.31 GiB] / in no VG: 0 [0   ]
+        pattern = re.compile(r"(?P<disk>.*)(?P<number>[\d]+)", re.M)
+        matched = find_groups_in_lines(pv_result, pattern)
```

Comment:
use `find_group_in_lines`, it returns only one, not a list.

Reply:
![image](https://github.com/microsoft/lisa/assets/26188281/9f7cb179-4e28-44ed-86a3-8cea505f67af)
It returns a list of dict

Reply:
It’s group, not groups

Reply:
Updated